### ISSUE TITLE
ID for fiscal inovices

### ISSUE THREAD

**Issue by gapi78** [InvoiceFox/InvoiceFox-superfolder#2]
I cant get id for the invoices which are fiscaly validated and then i cannot download pdf's

**Comment by InvoiceFox** [comment]
True, fiscal invoices use different numbering mechanism. I will add this, thank you!
